chunky/src/java/se/llbit/chunky/renderer/scene/sky/CelestialBodyType.java

```diff
@@ -0,0 +1,219 @@
+package se.llbit.chunky.renderer.scene.sky;
+
+import se.llbit.chunky.renderer.SceneIOProvider;
+import se.llbit.chunky.resources.BitmapImage;
+import se.llbit.chunky.resources.Texture;
+import se.llbit.json.JsonObject;
+import se.llbit.log.Log;
+import se.llbit.resources.ImageLoader;
+import se.llbit.util.Registerable;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+// TODO: use https://openjdk.org/jeps/409 sealed classes
```

Comment:
Maybe we shouldn't make this sealed so plugins in the future have the option to add their own celestial bodies